defaults/main.yml

```diff
@@ -1,4 +1,5 @@
 ---
+node_exporter_binary_local_dir: ''
```

Comment:
This should be below `node_exporter_version` to make it similar to other repositories in cloudalchemy.

Reply:
Fixed